agreement/common_test.go

```diff
@@ -529,8 +533,9 @@ func (v *voteMakerHelper) MakeRandomProposalValue() *proposalValue {
 
 func (v *voteMakerHelper) MakeRandomProposalPayload(t *testing.T, r round) (*proposal, *proposalValue) {
 	f := testBlockFactory{Owner: 1}
-	ve, err := f.AssembleBlock(r)
+	ue, err := f.AssembleBlock(r, nil)
```

Comment:
Do you recall why these end in `e`? Seems like `ub` is better here, and `pb` below.